### PR TITLE
fix the procurement-tenders table primary key

### PR DESCRIPTION
**Pull request by OriHoch**
fixed the procurement-tenders table primary key - which caused incorrect updating

**contains DB schema changes** - should drop the table, or modify the primary key manually

related issue: #76

### CODE REVIEW

File: budgetkey_data_pipelines/pipelines/procurement/tenders/parse_page_data.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from datetime import datetime
 
 TABLE_SCHEMA = {
-    "primaryKey": ["publication_id"],
+    "primaryKey": ["publication_id", "tender_type", "tender_id"],
```
Comment (akariv) [inline budgetkey_data_pipelines/pipelines/procurement/tenders/parse_page_data.py]: The reason why I removed tender id was that it sometimes got a `null` value (thus failing the processor).

This was the problematic tender: 
https://www.mr.gov.il/officestenders/Pages/officetender.aspx?pID=596701

Comment (akariv) [reply budgetkey_data_pipelines/pipelines/procurement/tenders/parse_page_data.py]: Perhaps "publication_id" & "tender_type" is enough?

Comment (OriHoch) [reply budgetkey_data_pipelines/pipelines/procurement/tenders/parse_page_data.py]: many central tenders have publication_id = 0 - and are identified by tender_id instead

Comment (OriHoch) [reply budgetkey_data_pipelines/pipelines/procurement/tenders/parse_page_data.py]: regarding the null value - looks like it was fixed

**Comment by akariv** [comment]
I think I already changed it back on master - pls check and rebase/close as necessary.

**Comment by OriHoch** [comment]
Cool, looks like it automatically closed when I rebased.. I'll verify that it's really merged

**Comment by OriHoch** [comment]
ok, looks like it's merged https://github.com/OpenBudget/budgetkey-data-pipelines/blob/master/datapackage_pipelines_budgetkey/pipelines/procurement/tenders/parse_page_data.py#L10

**Pull request closed without merging**